.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: ruby
 bundler_args: --without development
 before_install:
+  - gem update --system
   - gem install bundler
   - gem update bundler
   - bundle install --gemfile=Gemfile.ruby_dep --path vendor/bundle
 rvm:
   - 2.2.7
   - 2.3.4
   - 2.4.1
+  - 2.5.0
```

Comment:
@rymai next PR to be merged also could be this PR.
Right now the latest Ruby 2.5 version is `2.5.1`. You may be able to consider to adopt below way to use the latest version automatically.

```
    - 2.2
    - 2.3
    - 2.4
    - 2.5
```